pages/common/dolt-merge.md

```diff
@@ -0,0 +1,24 @@
+# dolt merge
+
+> Join two or more development histories together.
+> More information: <https://github.com/dolthub/dolt>.
+
+- Incorporate changes from the named commits into the current branch:
+
+`dolt merge {{branch_name}}`
+
+- Incorporate changes from the named commits (since the time their histories diverged from the current branch) into the current branch without updating the commit history:
```

Comment:
I think we shouldn't explain a merge in this page
```suggestion
- Incorporate changes from the named commits into the current branch without updating the commit history:
```

Reply:
I agree, solved in eb5c002ce1ccfd13c575dc75b90876ccb265adb1